### PR TITLE
Support `post_[bash,batch]_commands` post-processing scripting commands

### PR DESCRIPTION
**Pull request by UebelAndre**
closes https://github.com/bazelbuild/continuous-integration/issues/1688

### CODE REVIEW

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -1351,6 +1351,11 @@ def PrepareRepoInCwd(print_cmd_groups, initial_setup=False):
 
         execute_bazel_run(bazel_binary, platform, task_config.get("run_targets", None))
 
+        if platform == "windows":
```
Comment (meteorcloudy) [inline buildkite/bazelci.py:1354]: I think this should be moved to line 1520/1525, basically after the `index_targets`

**Review by meteorcloudy — CHANGES_REQUESTED** [review]
Thanks!

File: buildkite/bazelci.py (hunk 1 above)
Comment (UebelAndre) [reply buildkite/bazelci.py]: lol makes total sense. Moved!

**Comment by UebelAndre** [comment]
@meteorcloudy thanks for the review! When will this be available to use in other projects?

**Pull request merged**